setup/handlers.sh

```diff
@@ -0,0 +1,43 @@
+build() {
+  build_handler
+  docker compose build
+}
+
+build_all() {
+  build_handler
+  docker compose -f docker-compose.yml -f docker-compose-tools.yml build
+}
+
+start() {
+  echo '🚀 Starting blog with Docker 🚀'
+  echo ''
+  cd "$CURRENT_DIR" || exit
+  echo "◎ Starting blog with Docker..."
+  docker compose up -d nginx postgres php83 node fe elasticsearch
+}
+
+start_all() {
+  echo '🚀 Starting blog with Docker 🚀'
+  echo ''
+  cd "$CURRENT_DIR" || exit
+  echo "◎ Starting blog with Docker..."
+  docker compose -f docker-compose.yml -f docker-compose-tools.yml up -d
```

Comment:
@cslant/site-blog optional tools will be separated into the docker-compose-tools.yml file